.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: java
+jdk:
+  - openjdk8
+  - oraclejdk8
```

Comment:
I like that we're *not* turning on the oh-so-painfully-slow MacOS testing. Might miss a chance to catch things, but we have no intention of running this service under MacOS in production, and faster feedback on PRs will be nice.